flag.go

```diff
@@ -62,10 +66,11 @@ type Target struct {
 
 // An explanation is either a target or a rule or a prerequisite that wasn't met
 type Explanation struct {
-	Kind          string `json:"kind" bson:"kind"`
-	*Target       `json:"target,omitempty"`
-	*Rule         `json:"rule,omitempty"`
-	*Prerequisite `json:"prerequisite,omitempty"`
+	Kind                string `json:"kind" bson:"kind"`
+	*Target             `json:"target,omitempty"`
+	*Rule               `json:"rule,omitempty"`
+	*Prerequisite       `json:"prerequisite,omitempty"`
+	*VariationOrRollout `json:"fallthrough, omitempty"`
```

Comment:
Extra space in the annotation. Not sure if it matters, but probably want to clean it up.